rhoas-cli-kafka/README.adoc

```diff
@@ -243,13 +257,6 @@ After creating a Kafka instance, you can create Kafka topics to start producing
 [NOTE]
 ====
 You can use `rhoas kafka use` to switch to a specific Kafka instance.
-
-.Selecting a Kafka instance to use
-[source,shell]
-----
-$ rhoas kafka use --name my-kafka
-Kafka instance "my-kafka" has been set as the current instance.
```

Comment:
Did you mean to remove lines 248-252 entirely, or to split the command from the output?

Reply:
Yes; it seems redundant given the preceding line.